docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -316,70 +473,206 @@ Probe position will be reported in the `mosPCX`, `mosPCY` and `mosPCZ` global va
+
+### `G81` - DRILL CANNED CYCLE - FULL DEPTH
+
+```gcode
+; Usage
+G80 [Fnnn] [Rnnn] [Xnnn] [Ynnn] [Znnn]
+
+; At X=10 and Y=10, drill down at 500mm/min,
+; from Z=-5 to Z=-10, in one movement.
+; We then retract to Z=-5.
+G81 F500 R-5 Z-10 X10 Y10
+```
+
+Run a full-depth drilling cycle with _NO_ retraction. **WARNING**: - unless you are drilling very shallow holes, use a drilling cycle with retraction.
+
+The cycle will first move to the `R` height at the current location, then to the `X` and `Y` location if given.
+
+It will then move downwards towards `Z` at the given feed rate (or the previously used rate), reaching the `Z` location in one move. It will then retract to the `R` height and return.
+
+### `G83` DRILL CANNED CYCLE - PECK DRILLING WITH FULL RETRACT
+
+```gcode
+; Usage
+G83 [Fnnn] [Rnnn] [Qnnn] [Xnnn] [Ynnn] [Znnn]
+
+; At X=10 and Y=10, peck 1mm at a time at 500mm/min,
+; from Z=-5 to Z=-10, retracting to Z=-5 after each peck.
+G83 F500 R-5 Q1 Z-10 X10 Y10
+```
+
+Working in the same manner as `G73`, this cycle retracts to the initial Z position after each peck rather than retracting by the peck distance.
+
+This allows for easier chip clearing during the drilling cycle. If in doubt, use this cycle type for canned drilling as it is the least likely to break your drill if chip clearing is an issue.
+
+---
+
 ## Tool Management
 
 ### `M4000` - DEFINE TOOL
 
 ```gcode
+; Usage
 M4000 Pnn Rnn S"..." [Xnn] [Ynn]
+
+; Define tool index 5 as a 30mm diameter Face Mill
+M4000 P5 R15.0 S"30mm Face Mill 3 flute"
+
+; Define tool index 49 as a probe with 1mm tip radius and deflection values for X and Y
+M4000 P49 R1 S"Touch Probe" X0.05 Y0.01
 ```
 
 We need to store additional details about tools that RRF is not currently able to accommodate natively - this includes tool radius (`R`) and deflection values for probes (`X` and `Y`). Tool index is set using `P` and description is set using `S".."`. M4000` stores these custom values that allows us to use them, while configuring RRF with the relevant tool details using `M563`.
 
 ### `M4001` - FORGET TOOL
 
 ```gcode
+; Usage
 M4001 Pnn
+
+; Forget tool 19
+M4001 P19
 ```
 
-Remove the tool at index `Pnn`.
+Reset the tool at index `Pnn` to defaults.
 
 ### `T<N>` - EXECUTE TOOL CHANGE
 
 ```gcode
+; Usage
 Tnn
+
+; Change to tool 41
+T41
 ```
 
 This macro is built in to RRF, using the `t{free,pre,post}.g` files. If the target tool number is specified, then these files are executed in order. The operator is prompted to change to the correct tool and if this tool is a probe tool, will be asked to verify that the tool is connected by triggering it manually before proceeding.
 
 ---
 
+## Coolant Control
+
+### `M7.1` - ENABLE AIR BLAST
+
+```gcode
+; Usage
+M7.1
+```
+
+Enables the GPIO output associated with air-blast, set by the operator during the configuration wizard.
```

Comment:
Air blast doesn't seem to be a hyphenated word https://www.dictionary.com/browse/air-blast